FiveCalls/FiveCalls.xcodeproj/project.pbxproj

```diff
@@ -557,6 +604,7 @@
 				B512506921E6F762007D32DF /* FetchContactsOperation.swift */,
 				9C46B1F91E46EA58009B2C11 /* ReportOutcomeOperation.swift */,
 				A0D474B3202A22B90012BC94 /* FetchUserStatsOperation.swift */,
+				D07829742A7B32A8001EE5DB /* Operator.swift */,
```

Comment:
I think we can get rid of this now, right?

Reply:
Yep, gone